scripts/zones/Bastok_Markets/npcs/A.M.A.N._Reclaimer.lua

```diff
@@ -0,0 +1,17 @@
+-----------------------------------
+-- Area: Bastok Markets
+--  NPC: A.M.A.N._Reclaimer
+-- Records of Eminence NPC
+-----------------------------------
+
+function onTrade(player,npc,trade)
+end;
+
+function onTrigger(player,npc)
+end;
+
+function onEventUpdate(player,csid,option)
+end;
+
+function onEventFinish(player,csid,option)
+end;
```

Comment:
Please kill these `;`'s

Reply:
Since it's empty and we aren't adding any of the reclaimers for the other zones either, I'm just going to kill the file; I'm sure @Apocarypse won't mind.